services/frontend-service/src/ui/components/EnvironmentCard/EnvironmentCard.test.tsx

```diff
@@ -0,0 +1,108 @@
+/*This file is part of kuberpult.
+
+Kuberpult is free software: you can redistribute it and/or modify
+it under the terms of the Expat(MIT) License as published by
+the Free Software Foundation.
+
+Kuberpult is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+MIT License for more details.
+
+You should have received a copy of the MIT License
+along with kuberpult. If not, see <https://directory.fsf.org/wiki/License:Expat>.
+
+Copyright 2023 freiheit.com*/
+
+import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { EnvironmentGroup } from '../../../api/api';
+import { EnvironmentGroupCard } from './EnvironmentCard';
+
+const getNode = (group: EnvironmentGroup): JSX.Element | any => (
+    <MemoryRouter>
+        <EnvironmentGroupCard environmentGroup={group} />
+    </MemoryRouter>
+);
+const getWrapper = (group: EnvironmentGroup) => render(getNode(group));
+
+describe('Test env locks', () => {
```

Comment:
I think we should have a test for more than 1 group

Reply:
done (in EnvironmentsPage.test.tsx)